core/base/src/TMemberInspector.cxx

```diff
@@ -121,7 +121,7 @@ void TMemberInspector::GenericShowMembers(const char *topClassName, const void *
       }
    }
 
-   TClass *top = TClass::GetClass(topClassName);
+   TClass *top = TClass::GetClass(topClassName, true, isTransient);
```

Comment:
Maybe just add a comment so we remember not to remove the `isTransient` flag from the function call in the future

Reply:
It was not removed, just forgotten to be added when introduced :)